CryptoPrices/Sources/Home/Sources/Home/TrendingCoinsSectionView/TrendingCoinSection.swift

```diff
@@ -29,13 +29,12 @@ struct TrendingCoinSection: View {
             // TODO: - Remove dummy
             ScrollView(.vertical, showsIndicators: false) {
                 VStack {
-                    ForEach(0..<2) {_ in
+                    ForEach(0..<2) { _ in
                         TrendingCoinItemView()
                             .padding(.vertical, 8.0)
                             .onTapGesture {
-                                withAnimation {
-                                    homeState.didSelectCoin = true
-                                }
+                                // TODO: Remove hard-coded data
+                                homeState.didSelectCoin = "bitcoin"
```

Comment:
Oh this is intentional that we remove `withAnimation` and we don't need it anymore, right? 

When presenting the view, I can see that there is still a proper transition, even for the going back action, nice work 🎉

Reply:
Yeah, the `withAnimation` is no longer necessary because we don't manually add animation when transitioning from `Home` to `MyCoin` but let the `NavigationView` do it for us.